docs-client/src/containers/MethodPage/index.tsx

```diff
@@ -129,22 +129,16 @@ type Props = OwnProps &
   }>;
 
 const MethodPage: React.FunctionComponent<Props> = (props) => {
-  const service = props.specification.getServiceByName(
-    props.match.params.serviceName,
-  );
+  const params = props.match.params;
+  const service = props.specification.getServiceByName(params.serviceName);
   if (!service) {
     return <>Not found.</>;
   }
-
-  const method = service.methods.find(
-    (m) =>
-      m.name === props.match.params.methodName &&
-      m.httpMethod === props.match.params.httpMethod,
-  );
+  const id = `${params.serviceName}/${params.methodName}/${params.httpMethod}`;
+  const method = service.methods.find((m) => m.id === id);
```

Comment:
👍 Great to see this change, did we have issues with overlapping method names from different services in the previous version?

Reply:
> overlapping method names from different services in the previous version?

No, we didn't have such an issue. We have issues only with the same service. For example:
```java
class MyAnnotatedService {
    @Get("/foo)
    public HttpResponse foo() {...}

    @Get("/foo1)
    public HttpResponse foo(MyType type) {...}    <--- This isn't rendered.
}
```